vm/internal/backend/size.go

```diff
@@ -4,17 +4,18 @@ import (
 	"bytes"
 	"context"
 	"fmt"
-	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/api/types/container"
-	"github.com/docker/docker/client"
-	"github.com/felipecruz91/vackup-docker-extension/internal"
-	"github.com/felipecruz91/vackup-docker-extension/internal/log"
 	"io"
 	"os"
 	"path/filepath"
 	"runtime"
 	"strconv"
 	"strings"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/docker/volumes-backup-extension/internal"
+	"github.com/docker/volumes-backup-extension/internal/log"
```

Comment:
there are a lot of reformatting changes in this PR, could it be because the IDE of the original author (@felipecruz91 ?) is not correctly configured?